### DISCUSSION TITLE
Strange button behaviour when using an svg inside a button with wire:loading

### DISCUSSION THREAD

**Discussion by DonCamillo11** [livewire/livewire#8148] [7. Bugs (archived)]
### Livewire version

v3.4.9

### Laravel version

v10.48.3

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

1. Adding a loading indicator as svg to a button 
`<svg wire:loading.delay class="text-gray-200 dark:text-gray-100 fill-white animate-spin h-3 w-3 mr-2 mb-px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M12,1A11,11,0,1,0,23,12,11,11,0,0,0,12,1Zm0,19a8,8,0,1,1,8-8A8,8,0,0,1,12,20Z" opacity=".25"/><path d="M10.14,1.16a11,11,0,0,0-9,8.92A1.59,1.59,0,0,0,2.46,12,1.52,1.52,0,0,0,4.11,10.7a8,8,0,0,1,6.66-6.61A1.42,1.42,0,0,0,12,2.69h0A1.57,1.57,0,0,0,10.14,1.16Z" /></svg>`

### Repository or Wirebox URL

https://wirebox.app/b/4jnd9

### Additional information

After updating to the latest version there is a layout change for the buttons when an SVG is used as a loading indicator. The button temporarily becomes narrower before it returns to its old appearance. Please see the wirebox example.

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by joshhanley** [comment]
@DonCamillo11 your wirebox link is private so I can't see it. `wire:loading` uses `inline-block` by default, but an svg is possibly a `block` element. So try using `wire:loading.block.delay`. See here for details https://livewire.laravel.com/docs/wire-loading#customizing-css-display-property

**Reply by DonCamillo11** [reply]
The wirebox is now public. Unfortunately this doesn’t help. But maybe this is a Safari problem. In Firefox this doesn’t happen.
![Bildschirmfoto 2024-04-04 um 15 58 12](https://github.com/livewire/livewire/assets/125701444/f31d3687-1eff-47f4-857c-1d3a83b3b1da)
